tests/tests_discard_passphrase_keyfile.yml

```diff
@@ -0,0 +1,45 @@
+---
+- hosts: all
+
+  vars:
+    nbde_client_bindings:
+      - device: "{{ nbde_client_test_device }}"
+        keyfile: "{{ nbde_client_test_keyfile }}"
+        discard_passphrase: yes
+        servers:
+          - http://localhost
+      - device: "{{ nbde_client_test_device }}"
+        servers:
+          - http://localhost
+
+  tasks:
+    - name: Set up test environment
+      include: tasks/setup_test.yml
+
+    - name: Run the test
+      block:
+        - name: Use nbde_client role
+          include_role:
+            name: linux-system-roles.nbde_client
+
+        - name: Attempt to unlock device
+          include: tasks/verify_unlock_device.yml
+
+        - name: Make sure the attempt to unlock succeeded
+          assert:
+            that:
+              - not nbde_client_unlock.failed
+              - not nbde_client_close.failed
+
+        - name: Attempt to check whether default keyfile works
+          include: tasks/verify_default_keyfile.yml
+
+        - name: Make sure the default keyfile did not work
+          assert:
+            that:
+              - nbde_client_keyfile.rc != 0
```

Comment:
use `is success` similar to https://github.com/linux-system-roles/nbde_client/pull/3/files#r437623134